demo/lightning/checkpoint/multinode/train.py

```diff
@@ -48,6 +50,7 @@ def init_processes():
     os.environ["NODE_RANK"] = str(rank)
 
     configure_master_addr()
+    return rank
```

Comment:
So this function sets the environment variable "NODE_RANK". We don't need to return the rank value since we can just grab it from there (and we should probably keep a single source of truth)

Reply:
I updated the method to remove rank from the function return. I mostly agree with using a single source of truth for the rank value, but I'm trying to keep env var use centralized to help readability / maintainability.

Reply:
Yes, i think for anything that's not enforced by pytorch itself its better to return. We just don't have much control in this instance.